orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -61,9 +61,12 @@ def __init__(self, module):
         self._save_onnx_prefix = ''
 
         # Graph transformer config
+        # Specify cast propagation strategy. Currently two strategies are available, insert-and-reduce and flood-fill
```

Comment:
Is it possible to add a unit test on the frontend in ```orttraining_test_ortmodule_api.py```?

Reply:
This is only a graph transformation config setting, which will be used by the backend. We have already the initialization tests exercising this. What would be the test scenario? Do we wrap a model with ORTModule , set the graph transformation config and run training ? Or only read back the setting and verify?

Reply:
Apologies of putting this comment rather arbitrarily. I was asking more in terms of adding a unit test on the frontend to test the propagate and case op feature in this PR.